tech-specs/src/explore/design/ledger/pos-integration.md

```diff
@@ -0,0 +1,257 @@
+# PoS integration
+
+The [PoS system](/explore/design/pos.md) is integrated into Anoma ledger via:
+- an account with an internal address and a [native VP](vp.md#native-vps) and that validates any changes applied by transactions to the PoS account
+- transaction WASMs to perform various PoS actions, also available as a library code for custom made transactions
+
+The `votes_per_token` PoS system parameter must be chosen to satisfy the [Tendermint requirement](https://github.com/tendermint/spec/blob/60395941214439339cc60040944c67893b5f8145/spec/abci/apps.md#validator-updates) of `MaxTotalVotingPower = MaxInt64 / 8`.
+
+All [the data relevant to the PoS system](/explore/design/pos.md#storage) are stored under the PoS account's storage sub-space, with the following key schema (the PoS address prefix is omitted for clarity):
+
+- `parameters`: the system parameters
+- `parameters/{parameters_hash}`: the system parameters value with this `parameters_hash `
+- `parameters/{parameters_hash}/{validator_address}`: a set of validators who want to switch to parameters with this `parameters_hash` (only keys are needed here, so the value can be a `unit`)
```

Comment:
I think we should not track validator votes on parameter switches this way. Instead, we should use the n-party trade system - validators should have special accounts associated with their staked token amounts, and a 2/3 weighted majority of those accounts should be able to accept (probably by signature) a transaction to change proof-of-stake parameters (perhaps with a delay).

Reply:
I removed the extraneous storage keys for parameters updates for now. Let's follow-up on this with #351 and #352